config/webpack.config.js

```diff
@@ -347,6 +347,16 @@ module.exports = env => {
         .map(generateLandingPage),
     );
 
+    // Create a placeholder home page.
+    baseConfig.plugins.push(
+      new HtmlPlugin({
+        filename: path.join(outputPath, '..', 'index.html'),
+        inject: false,
+        title: 'VA.gov',
+      }),
+    );
```

Comment:
Needed to put a file at the home page so `start-server-and-test` in the CircleCI Cypress script can use it to detect when the server is running.